runmanager/__main__.py

```diff
@@ -697,7 +696,7 @@ def __init__(self, tabWidget, globals_file, group_name):
 
         loader = UiLoader()
         loader.registerCustomWidget(TableView)
-        self.ui = loader.load('group.ui')
+        self.ui = loader.load(os.path.join(runmanager_dir, 'group.ui'))
```

Comment:
Why not `str(runmanager_dir / 'group.ui')` and (equivalent) elsewhere?

Reply:
Unless I'm unaware of a Python feature, you can't divide strings! `os.path.join` also ensures the use of `os.sep` so that paths are constructed properly for the current runtime platform.

Reply:
This PR sees `runmanager_dir` as type `pathlib.Path`, which supports path concatenation via the `/` operator, the `pathlib` equivalent of (and obviating the need for) `os.path.join`.

Reply:
Fair enough, I just didn't want to mix Path and strings too much - will probably go through and mass convert come other time. I'll change this one but there are plenty more where that came from!

The even better solution is for `loader.load()` to accept a pathlike object :p